cetmix_tower_server/views/cx_tower_plan_line_view.xml

```diff
@@ -38,8 +42,53 @@
                     <notebook>
                         <page name="preview" string="Command Preview">
                             <group>
-                                <field name="command_code" />
+                                <field name="action" />
+                                <field
+                                    name="flight_plan_id"
+                                    string="Flight Plan"
+                                    attrs="{'invisible': [('action', '!=', 'plan')]}"
+                                />
                             </group>
+                                <field
+                                name="command_code"
+                                attrs="{'invisible': [('action', 'not in', ['ssh_command', 'python_code'])]}"
+                            />
+                                <field
+                                name="file_template_id"
```

Comment:
Why do you put this field here?
Also there is no template code preview available in the form:
![image](https://github.com/user-attachments/assets/8202fa3a-f2d9-4c21-acf6-e2836b0d57c3)